### PR TITLE
chore: adjust release procedure to account for beta releases

### PR DESCRIPTION
**Pull request by larsmoa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,7 +167,13 @@ If you are not currently logged into npm on your CLI, do the following:
...
 
 ```bash
-yarn release
+yarn release:prod
+```
+
+or if you are releasing a beta version:
+
+```bash
+yarn release:beta
```
Comment (maksnester) [inline README.md:176]:
```suggestion
yarn release --tag=beta
```

File: README.md [hunk 2]
```diff
@@ -181,17 +187,16 @@ targeting the master branch.
...
 Also, you can check what's committed from the previous tag with that command:
    ```bash
-   git log --pretty=format:"%s <%an> – %h" @cognite/reveal@1.2.0...HEAD
+   git log --pretty=format:"* %s" @cognite/reveal@2.0.0...HEAD
    ```
    Use the following template:
    ```md
-   This version of Reveal depends on ThreeJS <version>. Your application should match
-   the version used to ensure compatibility. 
+   This is a beta release of the next major version of Reveal. Reveal 2.0 comes with ThreeJS embedded so you do not to have this as a dependency in your project. If you still want to have it as a direct dependency, it must match the version used by Reveal (r<version>).
```
Comment (maksnester) [inline README.md:199]:
```suggestion
   This is a beta release of the next major version of Reveal. Reveal 2.0 comes with ThreeJS embedded so you do not have this as a dependency in your project. If you still want to have it as a direct dependency, it must match the version used by Reveal (r<version>). You can also use three.js version exported by Reveal as `import { THREE } from '@cognite/reveal`.
```

File: parser-worker/package.json [hunk 3]
```diff
@@ -34,7 +34,8 @@
     "bump:commit": "cross-var git commit -am \"chore(parser-worker): bump version $npm_package_version\"",
     "bump:tag": "cross-var git tag $npm_package_name@$npm_package_version",
     "bump:push": "git push -u origin head && git push --tags",
-    "release": "yarn build:prod && yarn publish --non-interactive"
+    "release:prod": "yarn build:prod && yarn publish --non-interactive",
+    "release:beta": "yarn build:prod && yarn publish --non-interactive --tag beta"
```
Comment (maksnester) [inline parser-worker/package.json:38]:
```suggestion
    "release": "yarn build:prod && yarn publish --non-interactive"
```

File: viewer/package.json [hunk 4]
```diff
@@ -30,7 +30,8 @@
     "bump:commit": "git add -A && cross-var git commit -m \"chore(viewer): bump version $npm_package_version\"",
     "bump:tag": "cross-var git tag $npm_package_name@$npm_package_version",
     "bump:push": "git push -u origin head && git push --tags",
-    "release": "yarn build:prod && cd dist && yarn publish --non-interactive"
+    "release:prod": "yarn build:prod && cd dist && yarn publish --non-interactive",
+    "release:beta": "yarn build:prod && cd dist && yarn publish --non-interactive --tag beta"
```
Comment (maksnester) [inline viewer/package.json:34]:
```suggestion
    "release": "yarn build:prod && cd dist && yarn publish --non-interactive"
```

**Review by maksnester — APPROVED** [review]
@larsmoa Readme changes look good, but I suggest not adding unnecessary build scripts. You can simply pass the CLI `--tag=beta` argument directly to `yarn release`.

File: README.md (hunk 2 above)
Comment (larsmoa) [reply README.md:199]: Aha - great!!

**Pull request merged**